process/incident_management.md

```diff
@@ -0,0 +1,273 @@
+
+### Incident Technical Lead
+* The incident technical lead is by default the First Responder
+* An Incident Technical Lead **can be changed only by an __explicit decision__ of the Incident Commander.**
+  * If you’re a First Responder, but don’t want to be the Lead, you must coordinate that change with the Incident Commander.
+
+#### Responsibilities
+The technical lead of an incident is in charge of leading the technical investigation of the incident and make sure that it is recovered as quickly as possible.
+
+If the incident is not resolved at the end of the current shift then the Incident Tech Lead role gets handed over to the next shift, while Incident Owner is not.
+
+The Incident Tech Lead is responsible to contribute to the documentation of what happened in the RCA document and help the Incident Owner (if not self) to close any gaps in the overall management of the incident
+
+Summary:
+* Is in charge of the investigation and recovery
+* Role is handed over from one shift to the other until full resolution
+* Helps the Incident Owner document the RCA and PMR Action Items.
+
+### Supporting SRE / Parallel Investigator
+A parallel investigator is someone who will be involved hands-on in investigating and helping the fast recovery of the incident.
+A parallel investigator will get involved either by the Incident Commander or by any other members of the slack channel.
+
+#### Responsibilities
+The investigators have the responsibility to describe the action they are taking in the dedicated slack channel to ensure that there is awareness on the steps being taken for the recovery of everyone involved in the process. This can also be done by someone else assisting in the investigation while the main investigator is busy with the recovery.
+
+### Incident Commander
+
+The key element for this role is to ensure that the Incident Commander is solely focused on "oiling the machinery" of incident recovery and will not participate in the hands-on resolution. In Fact the main goal for the Incident Commander is to deal with the overall coordination overhead of the incident recovery, with a cool head, while the investigators are focused 100% on performing the recovery. This will allow faster recovery, better communication flows and less errors during the stress of the incident management.
+
+The Incident Commander can elect someone to be the "scribe" during the incident. The duty of the scribe is to report in the RCA document all the important information (technical details, timeline, ...) while the Incident Commander is focused on helping with the coordination.
+
+#### Responsibilities
+The coordination work under the responsibility of the Incident Commander is as follows:
+1. If the incident has been uncovered by SRE and doesn't have an existing "external" ticket (Jira or ICM for ARO) then the Incident Commander will create a Jira ticket to track the overall life of the incident.
```

Comment:
remove ref of Jira/ICM and just mention a "tracking ticket"

Reply:
updated in [f799548](https://github.com/operate-first/sre/pull/17/commits/f79954870cd4360b9ede03ca11adbd53bd87232d)